src/System.Security.Cryptography.Cng/tests/ImportExportTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class ImportExportTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
The test project is marked with UnsupportedPlatforms. I was following the *.Csp test code pattern here.